src/Daemon/DaemonConfiguration.h

```
@@ -340,6 +340,265 @@ namespace {
+            updated += 1;
+          }
+          catch(std::exception& e)
+          {
+            // pass
+          }
+        }
+        else if (cfgKey.compare("add-exclusive-node") == 0)
+        {
+          
+          exclusiveNodes.push_back(cfgValue);
+          config.exclusiveNodes = exclusiveNodes;
+          updated += 1;
+        }
+        else if (cfgKey.compare("add-peer") == 0)
+        {
+          //config.peers = j["add-peer"].get<std::vector<std::string>>();
+          peers.push_back(cfgValue);
+          config.peers = peers;
+          updated += 1;
+        }
+        else if (cfgKey.compare("add-priority-node") == 0)
+        {
+          priorityNodes.push_back(cfgValue);
+          config.priorityNodes = priorityNodes;
+          updated += 1;
+        }
+        else if (cfgKey.compare("seed-node") == 0)
+        {
+          seedNodes.push_back(cfgValue);
+          config.seedNodes = seedNodes;
+          updated += 1;
+        }
+        else if (cfgKey.compare("enable-blockexplorer") == 0)
+        {
+          config.enableBlockExplorer =  cfgValue.at(0) == '1' ? true : false;
+          updated += 1;
+        }
+        else if (cfgKey.compare("enable-cors") == 0)
+        {
+          cors.push_back(cfgValue);
+          config.enableCors = cors;
+          updated += 1;
+        }
+        else if (cfgKey.compare("fee-address") == 0)
+        {
+          config.feeAddress = cfgValue;
+          updated += 1;
+        }
+        else if (cfgKey.compare("fee-amount") == 0)
+        {
+          try
+          {
+            config.feeAmount = std::stoi(cfgValue);
+          }
+          catch(std::exception& e)
+          {
+            // pass
+          }
+        }
+        else
+        {
+          for (auto c: cfgKey) 
+          {
+            if (static_cast<unsigned char>(c) > 127) 
+            {
+                throw std::runtime_error(std::string("Bad/invalid config file"));
```

You don't need to create a string here, just `throw std::runtime_error("Bad/invalid config file");` works.

---

yes sir